app/appserver/templates/base.html

```diff
@@ -1,87 +1,40 @@
-<%!
-# retrieve settings
-isLite = (cherrypy.config['product_type'] == 'lite' or cherrypy.config['product_type'] == 'lite_free')
-
-if cherrypy.config['product_type'] == 'hunk':
-    faviconFile = 'favicon_hunk.ico'
-elif isLite:
-    faviconFile = 'favicon_lite.ico'
-else:
-    faviconFile = 'favicon.ico'
-
-faviconFilePath = '/static/img/' + faviconFile
-
-config_qs = dict(autoload=1)
-if hasattr(cherrypy.request, 'embed') and cherrypy.request.embed:
-    config_qs['embed'] = 1
-%>\
-
-<%
-app_name = app
-
-splk_root = "/static"
-app_root = "/".join([splk_root,"app",app_name])
-
-app_js = app_root + '/js'
-
-app_js_build = app_root + '/js/build'
-app_css_build = app_root + '/css'
-
-pageJSName = page+'_page'
-
-# optimized js and css
-common_css = "%s/common.css" % app_css_build
-page_css ="%s/%s.css" % (app_css_build, page)
-bootstrap_enterprise_css = "%s/bootstrap-enterprise.css" % app_css_build
-
-page_js = "%s/%s.js" % (app_js_build ,pageJSName)
-%>\
-
-<!doctype html>
-<!--[if lt IE 7]> <html lang="${i18n.current_lang()[0]|h}" class="no-js ie  lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>    <html lang="${i18n.current_lang()[0]|h}" class="no-js ie7 lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>    <html lang="${i18n.current_lang()[0]|h}" class="no-js ie8 lt-ie9"> <![endif]-->
-<!--[if IE 9]>    <html lang="${i18n.current_lang()[0]|h}" class="no-js ie9"> <![endif]-->
-<!--[if gt IE 9]><!-->
-<html lang="${i18n.current_lang()[0]|h}" class="no-js"> <!--<![endif]-->
+<!--
+  ~ Copyright 2021 Splunk Inc.
```

Comment:
This is an AOB-generated file. It is not advisable to change this.

Reply:
please remove this comment from generated file immediately. all files in https://github.com/databrickslabs, unless vendored or otherwise specified, MUST have Databricks license and no other one.

Reply:
Done